curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65f3fa60a93b84110b3f1708.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 65f3fa60a93b84110b3f1708
+title: Step 10
+challengeType: 20
+dashedName: step-10
+---
+
+# --description--
+
+You want to return a meaningful string representation. For example, in case of `v1`, you want to return a tuple with the values of the vector components: `(2, 3)`.
```

Comment:
Have we taught the campers anywhere in the curriculum how to return multiple items using `return f'{self.x, self.y}' `?

Reply:
What do you mean with multiple items? The method should return a single string.